src/core/mesh/qgsmeshtracerenderer.cpp

```diff
@@ -299,9 +307,10 @@ void QgsMeshStreamField::updateSize( const QgsRenderContext &renderContext )
     return;
   }
 
-
   QgsRectangle fieldInterestZoneInDeviceCoordinates = QgsMeshLayerUtils::boundingBoxToScreenRectangle( deviceMapToPixel, interestZoneExtent );
-  mFieldTopLeftInDeviceCoordinates = QPoint( int( fieldInterestZoneInDeviceCoordinates.xMinimum() ), int( fieldInterestZoneInDeviceCoordinates.yMinimum() ) );
+  mFieldTopLeftInDeviceCoordinates =
+    QPoint( static_cast<int>( fieldInterestZoneInDeviceCoordinates.xMinimum() + 0.5 ),
+            static_cast<int>( fieldInterestZoneInDeviceCoordinates.yMinimum() + 0.5 ) );
```

Comment:
make clang-tidy happy?
```suggestion
    QPoint( static_cast<int>( lround( fieldInterestZoneInDeviceCoordinates.xMinimum() ) ),
            static_cast<int>( lround( fieldInterestZoneInDeviceCoordinates.yMinimum() ) ) );
```